java-cfenv-boot/src/test/java/io/pivotal/cfenv/spring/boot/CassandraCfEnvProcessorTests.java

```diff
@@ -30,38 +30,45 @@
  */
 public class CassandraCfEnvProcessorTests extends AbstractCfEnvTests {
 
+	public static final String SPRING_CASSANDRA = "spring.cassandra";
```

Comment:
Nitpicking I know 😅  but that public seems odd.

Reply:
Same for other constants introduced in tests.

Reply:
ah true, good catch - I guess it was useful before I re re factored...